plugins/wake/wake.plugin.sh

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env bash
+# oh-my-bash.module: wake-on-lan wrapper + autocompletion
+# wake.plugin.sh
+# Author: hoek from 0ut3r.space
+# Based on oh-my-zsh wakeonlan plugin @ commit 1d9eacb34f59f3bf82a9de0d7b474cb4c501e3fd
+
+function wake {
+  local cfgdir=~/.wakeonlan
+  local cfgfile=$cfgdir/$1
+
+  if [[ ! $1 || ! -f $cfgfile ]]; then
+    _omb_util_print 'Usage: wake <device>'
+    if [[ -d $cfgdir ]]; then
+      local devices
+      devices=$(ls "$cfgdir" 2>/dev/null | tr '\n' ' ')
+      _omb_util_print "Available devices: $devices"
```

Comment:
The CI produces the error on the output example in `README`. The root cause is that the ending newline is converted to ` ` here. We should probably connect the filenames with a space explicitly using the expansion `${array[*]}`.

```suggestion
      _omb_util_split devices "$(ls -- "$cfgdir" 2>/dev/null)" $'\n'
      local IFS=$' \t\n'
      _omb_util_print "Available devices: ${devices[*]}"
```